biothings/hub/databuild/builder.py

```diff
@@ -833,17 +833,14 @@ async def merge_source(self, src_name, batch_size=100000, ids=None, job_manager=
         got_error = False
         # grab ids only, so we can get more, let's say 10 times more
         id_batch_size = batch_size * 10
+
+        # FIXME id_provider initialized below will be overwritten by `if _query and ids is None:` code block
         if ids:
-            self.logger.info(
-                "Merging '%s' specific list of _ids, create merger job with batch_size=%d" % (src_name, batch_size)
-            )
+            self.logger.info("Merging '%s' specific list of _ids, create merger job with batch_size=%d" % (src_name, batch_size))
```

Comment:
@erikyao let's change it to this while you are fixing its formatting?
```
self.logger.info("Merging '%s' specific list of _ids, create merger job with batch_size=%d", src_name, batch_size)
```
same for a few other log lines below